include/openPMD/binding/c/Series.h

```diff
@@ -0,0 +1,137 @@
+#ifndef OPENPMD_SERIES_H
+#define OPENPMD_SERIES_H
+
+#include <openPMD/config.hpp>
+
+#include <openPMD/binding/c/Container_Iteration.h>
+#include <openPMD/binding/c/IO/Access.h>
+#include <openPMD/binding/c/IterationEncoding.h>
+#include <openPMD/binding/c/ReadIterations.h>
+#include <openPMD/binding/c/WriteIterations.h>
+#include <openPMD/binding/c/backend/Attributable.h>
+
+#if openPMD_HAVE_MPI
+#include <mpi.h>
+#endif
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    typedef struct openPMD_Series openPMD_Series;
+
+    // returns a non-owning pointer
+    const openPMD_Attributable *
+    openPMD_Series_getConstAttributable(const openPMD_Series *series);
+
+    // returns a non-owning pointer
+    openPMD_Attributable *
+    openPMD_Series_getAttributable(openPMD_Series *series);
+
+    openPMD_Series *openPMD_Series_new();
+
+#if openPMD_HAVE_MPI
```

Comment:
For the C++ API we're planning to split headers into parallel and non-parallel headers which would simplify shipping openPMD-api since you don't need separate MPI and non-MPI builds.
It would probably be good to start out with such a separation in the C bindings from the beginning already.

Reply:
I'll think about it. My main use case is calling openPMD_api from Julia, and there we always have MPI available.